homeassistant/components/sems/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "sems",
+  "name": "SEMS Portal",
+  "codeowners": ["@andrewgierens"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/sems",
+  "homekit": {},
```

Comment:
sorry missed that one: please remove that empty key

Reply:
No worries - Done :)